index.html

```diff
@@ -443,12 +443,11 @@ <h3>
           </li>
           <li>Let <var>billingAddress</var> be null.
           </li>
-          <li>If <var>request</var>.<a data-cite=
+          <li>If <var>request</var> <a data-cite=
           "payment-request#dfn-options">[[\options]]</a>["<a data-cite=
-          "payment-request#dom-paymentoptions-requestbillingaddress"><code>requestBillingAddress</code></a>"]
+          "payment-request#dom-paymentoptions-requestshipping"><code>requestShipping</code></a>"]
           is true:
             <ol>
-              <li>
```

Comment:
This li needs to be here, or I think the ol becomes invalid html.

Reply:
As a syntax I think there is no problem regardless of the location ofl li. and I think ol could be  a below of <div> tag for note. Anyway when I test previous code with Chrome browser that was not show 1 in ordered list. that's why I tried to change like this. But in case of firefox it show 1 as an ordered list.
Anyway I reverted this change you suggested :)